lms/djangoapps/courseware/views/index.py

```diff
@@ -463,6 +463,7 @@ def _create_courseware_context(self, request):
             'disable_accordion': hide_course_navigation,
             'disable_footer': True,
             'show_search': show_search,
+            'course_experience_mode': course_experience_mode
```

Comment:
Let's pass the the icon class name, instead of writing the logic in the template